hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/specifications/TargetSpecifications.java

```diff
@@ -148,6 +150,36 @@ private TargetSpecifications() {
         };
     }
 
+    /**
+     * {@link Specification} for retrieving {@link Target}s by "like attribute
+     * value".
+     *
+     * @param searchText
+     *            to be filtered on
+     * @return the {@link Target} {@link Specification}
+     */
+    public static Specification<JpaTarget> likeAttributeValue(final String searchText) {
+        return (targetRoot, query, cb) -> {
+            final String searchTextToLower = searchText.toLowerCase();
+            final MapJoin<JpaTarget, String, String> attributeMap = targetRoot.join(JpaTarget_.controllerAttributes,
+                    JoinType.LEFT);
+            query.distinct(true);
+            return cb.or(cb.like(cb.lower(attributeMap.value()), searchTextToLower));
```

Comment:
The "cb.or" at the beginning of the return statement is not needed. Please remove it. Thanks

Reply:
Done. I've also removed the whitespace changes.